my-dsl.md

```diff
@@ -1,38 +1,40 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
-
+The name of the language is django.
+[Django Project](https://www.djangoproject.com/)
 
 # Domain
 _Describe the language's domain in five words._
-
+Web development
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
-
+When someone executes a program using Django, it is run as a Python program. When certain requests are made, the request is converted to a series of SQL statements that alter a database to follow a data model.
```

Comment:
Can you describe it in more depth? What are the high level steps for accepting requests? How does Django use MVC (model-view-controller)?